packages/suite/src/support/suite/ConnectedIntlProvider.tsx

```diff
@@ -8,7 +8,17 @@ const ConnectedIntlProvider: React.FC = ({ children }) => {
         messages: state.suite.messages,
     }));
     return (
-        <IntlProvider locale={locale} messages={messages}>
+        <IntlProvider
+            locale={locale}
+            messages={messages}
+            onError={err => {
```

Comment:
default error handler using `process.env.NODE_ENV !== 'production'` condition, see https://github.com/formatjs/formatjs/blob/930ed77c458bf740f6863839ad9d2f3c820b71c3/packages/intl/src/utils.ts#L29

So if you override it, please add that condition too...

Reply:
ok. updated

Reply:
We use `import { isDev } from '@suite-utils/build';` and not process.env.NODE_ENV directly. We could discuss if it's useful, but let's keep it consistent.

Reply:
sorry for nitpicking

Reply:
I know we do :D but from your previous reaction I thought you want it to be `import { isDev } from '@suite-utils/build';` literally :D

Reply:
updated